specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizerReceipt.json

```diff
@@ -0,0 +1,266 @@
+        "parameters": [
+          {
+            "$ref": "Ocr.json#/parameters/ImageStream"
+          }
+        ],
+        "consumes": [
+          "application/octet-stream"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "202": {
+            "description": "The service has accepted the request and will start processing later. It will return 'Accepted' immediately and include an 'Operation-Location' header. Client side should further query the operation status using the URL specified in this header. The 'Operation-Location' URL will expire in 48 hours.",
+            "headers": {
+              "Operation-Location": {
+                "description": "URL to query for status of the operation. The URL will expire in 48 hours. ",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "Ocr.json#/definitions/ComputerVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful 'Batch Read Receipt' request": {
+            "$ref": "./examples/SuccessfulBatchReadReceiptWithStream.json"
+          }
+        }
+      }
+    }
+  },
+
+  "definitions": {
+    "readReceiptResult": {
+      "type": "object",
+      "description": "Analysis result of the 'Batch Read Receipt' operation.",
+      "properties": {
+        "status": {
+          "description": "Status of the read operation.",
+          "$ref": "Ocr.json#/definitions/OperationStatus"
+        },
+        "recognitionResults": {
+          "description": "Text recognition result of the 'Batch Read Receipt' operation.",
+          "type": "array",
+          "items": {
+            "$ref": "Ocr.json#/definitions/TextRecognitionResult"
+          }
+        },
+        "understandingResults": {
+          "description": "Semantic understanding result of the 'Batch Read Receipt' operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/understandingResult"
+          }
+        }
+      }
+    },
+
+    "understandingResult": {
+      "properties": {
+        "pages": {
+          "description": "List of pages where the document is found.",
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "x-nullable": false
+          }
+        },
+        "fields": {
+          "description": "Dictionary of recognized field values.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/fieldValue"
+          }
+        }
+      }
+    },
+
+    "elementReference": {
+      "description": "Reference to an OCR word.",
+      "properties": {
+        "$ref": {
+          "type": "string"
+        }
+      }
+    },
+
+    "fieldValue": {
+      "description": "Base class representing a recognized field value.",
```

Comment:
Please check which of the values are readonly and set these accordingly

Reply:
To my knowledge, Swagger 2.0 that AutoRest uses does not support marking properties as readonly.

Reply:
2.0 does support readonly. please refer example [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json#L241)

Reply:
@ramparab was this addressed?

Reply:
I have captured this feedback in feedback.receipt.md. @bojunehsu to address this in future updates.